app/models/miq_report.rb

```diff
@@ -31,7 +31,7 @@ class MiqReport < ApplicationRecord
   serialize :display_filter
 
   validates_presence_of     :name, :title, :db, :rpt_group
-  validates_uniqueness_of   :name
+  validates_uniqueness_of   :name, :unless => :seeding?
```

Comment:
```suggestion
  validates_uniqueness_of   :name, :unless => :seeding
```

seems simpler and we can then dump the `def seeding?`